data/decks/hyde.json

```diff
@@ -0,0 +1,205 @@
+{
+	"id": "hyde",
+	"season": 6,
+	"display_name": "Hyde Kido",
+	"cardback": "cardback_hyde.jpg",
+	"has_sealed_area": true,
+	"exceed_cost": 1,
+	"character_action_default": [{
+		"gauge_cost": 0,
+		"force_cost": 0,
+		"effect": {
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "close", "amount": 1 },
+				{ "effect_type": "retreat", "amount": 1 }
+			],
+			"and": {
+				"condition": "min_cards_in_hand",
+				"condition_amount": 1,
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "strike_faceup"},
+					{
+						"effect_type": "gauge_from_hand",
+						"min_amount": 1,
+						"max_amount": 1
+					}
+				]
+			}
+		}
+	}],
+	"character_action_exceeded": [
+		{
+			"gauge_cost": 1,
+			"force_cost": 0,
+			"effect": {
+				"effect_type": "strike_from_gauge"
+			}
+		},
+		{
+			"gauge_cost": 0,
+			"force_cost": 0,
+			"min_hand_size": 1,
+			"action_name": "Revert",
+			"effect": {
+				"effect_type": "gauge_from_hand",
+				"min_amount": 1,
+				"max_amount": 1,
+				"and": {
+					"effect_type": "revert",
+					"and": {
+						"effect_type": "take_bonus_actions",
+						"amount": 1
+					}
+				}
+			}
+		}
+	],
+	"ability_effects": [
+	],
+	"exceed_ability_effects": [
+		{
+			"timing": "during_strike",
+			"condition": "used_character_action",
+			"condition_details": ["exceed", 0],
```

Comment:
they only have the exceed_ability_effects when exceeded, so why the need for "exceed" in the condition details?

Reply:
The specific issue I ran into is that revert -> exceed with bonus action -> strike tricked it into thinking you were using Hyde's normal exceed UA, so I wanted to be able to specify exactly which ability it was looking for; including "exceed" vs "default" was mostly just future-proofing but I can reduce it to just the index if you don't think it'd ever be relevant

Reply:
I see. I guess that makes sense.
What we should probably do is be able to set arbitrary flags from the JSON and check those flags as a condition.
However, for simplicity maybe just go with what you've got for now.